zkevm-circuits/src/super_circuit.rs

```diff
@@ -188,6 +200,12 @@ impl<F: Field, const MAX_TXS: usize, const MAX_CALLDATA: usize> Circuit<F>
         config
             .evm_circuit
             .assign_block(&mut layouter, &self.block)?;
+        config.state_circuit.assign(
+            &mut layouter,
+            &self.block.rws.table_assignments(),
+            self.block.state_circuit_pad_to,
+            self.block.randomness,
```

Comment:
We can make the order of these arguments the same as the ones for RW table.. not a big issue
```suggestion
            self.block.randomness,
            self.block.state_circuit_pad_to,
```

Reply:
yea, changed the order in state circuit. seems more intuitive